malos/driver.proto

```diff
@@ -166,3 +166,37 @@ message LedValue {
 message EverloopImage {
   repeated LedValue led = 1;
 }
+
+
+/* ZigBee messages */
+
+// Network start event
+message ZigBeeNetworkStatus {
+  bool is_active = 1;
+  uint32 pan_id = 2;
```

Comment:
Please add comments for fields. What is pan_id ? If possible, add a longer name for the field (without abbreviations).